php_phongo.c

```diff
@@ -963,7 +963,7 @@ void php_phongo_read_preference_to_zval(zval *retval, const mongoc_read_prefs_t
 {
 	const bson_t *tags = mongoc_read_prefs_get_tags(read_prefs);
 
-	array_init_size(retval, 2);
+	array_init_size(retval, 3);
```

Comment:
This just controls the initial HashTable size, so it should be fine. The buckets are still empty until we fill them.